### DISCUSSION TITLE
Error while running end2end_example/cybersecurity/3-build-accelerator-with-finn.ipynb

### DISCUSSION THREAD

**Discussion by guptamrinal** [Xilinx/finn#581] [Q&A]
I'm facing the following error while executing 10th step in 3-build-accelerator-with-finn.ipynb.The error comes during "Running step: step_set_fifo_depths [11/17]"

 FileNotFoundError: [Errno 2] No such file or directory: '/tmp/finn_dev_nick/vivado_stitch_proj_kbkt75vx/all_verilog_srcs.txt'

**Comment by fpjentzsch** [comment]
Hi, this might be the same issue as here: https://github.com/Xilinx/finn/discussions/497

Did you apply the Y2K22 patch to Vivado? (https://support.xilinx.com/s/article/76960?language=en_US)
